### PR TITLE
Backface Culling Additions

### PR DESCRIPTION
**Pull request by KeflerExe**
Added "SetBackfaceCulling()" at "vtkF3DRenderer::ConfigureActorsProperties()".
Added  "BackfaceCullingVisible" in class "vtkF3DRenderer".
Added CheatSheet text at "vtkF3DRenderer::FillCheatSheetHotkeys" for backface culling.

### CODE REVIEW

File: vtkext/private/module/vtkF3DRenderer.cxx [hunk 1]
```diff
@@ -1315,6 +1315,7 @@ void vtkF3DRenderer::FillCheatSheetHotkeys(std::stringstream& cheatSheetText)
   cheatSheetText << std::fixed;
   cheatSheetText << " L: Light (increase, shift+L: decrease) [" << this->LightIntensity << "]"
                  << " \n";
+  cheatSheetText << " F1: Enable Backface Culling " << (this->BackfaceCullingVisible ? "[ON]" : "[OFF]") << "\n";
```
Comment (mwestphal) [inline vtkext/private/module/vtkF3DRenderer.cxx:1318]: lets not add a interactiive binding for this, I think it is too specific.

Comment (KeflerExe) [reply vtkext/private/module/vtkF3DRenderer.cxx]: Fixed!

File: application/F3DOptionsParser.cxx [hunk 2]
```diff
@@ -345,6 +345,7 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     this->DeclareOption(grp2, "point-type", "", "Point sprites type when showing point sprites", options.getAsStringRef("model.point-sprites.type"), HasDefault::YES, MayHaveConfig::YES, "<sphere|gaussian>");
     this->DeclareOption(grp2, "point-size", "", "Point size when showing vertices or point sprites", options.getAsDoubleRef("render.point-size"), HasDefault::YES, MayHaveConfig::YES, "<size>");
     this->DeclareOption(grp2, "line-width", "", "Line width when showing edges", options.getAsDoubleRef("render.line-width"), HasDefault::YES, MayHaveConfig::YES, "<width>");
+    this->DeclareOption(grp2, "backface", "", "Backface default (visible) or hidden", options.getAsStringRef("render.backface"), HasDefault::YES, MayHaveConfig::YES, "<default|hidden>");
```
Comment (mwestphal) [inline application/F3DOptionsParser.cxx:348]: I still think default should be unchanged.

possible value:

 - default
 - visible
 - hidden

**Comment by mwestphal** [comment]
hey @KeflerExe 

I'll take this over if that's okay.

**Comment by mwestphal** [comment]
https://github.com/f3d-app/f3d/pull/1480

**Pull request closed without merging**